packages/graph/src/Graph.ts

```diff
@@ -182,75 +182,116 @@ export class Graph extends SVGZoomWidget {
         return this;
     }
 
-    enter(domNode, element) {
-        super.enter(domNode, element);
-        const context = this;
+    //  Drag  ---
+    private _neighborOffsets: Array<{ neighbor: Vertex, offsetX: number, offsetY: number }> = [];
+    dragstart(d) {
+        if (this.allowDragging()) {
+            d3Event.sourceEvent.stopPropagation();
 
-        //  Drag  ---
-        function dragstart(d) {
-            if (context.allowDragging()) {
-                d3Event.sourceEvent.stopPropagation();
-                d.__drag_dx = d3Event.x - d.x();
-                d.__drag_dy = d3Event.y - d.y();
-                context._dragging = true;
-                if (context.forceLayout) {
-                    if (!d3Event.active) context.forceLayout.force.alphaTarget(0.3).restart();
-                    const forceNode = context.forceLayout.vertexMap[d.id()];
-                    forceNode.fixed = true;
-                    forceNode.fx = forceNode.x;
-                    forceNode.fy = forceNode.y;
-                }
-                if (Platform.svgMarkerGlitch) {
-                    context._graphData.nodeEdges(d.id()).forEach(function (id) {
-                        const edge = context._graphData.edge(id);
-                        context._pushMarkers(edge.element());
-                    });
-                }
+            d.__drag_dx = d3Event.x - d.x();
+            d.__drag_dy = d3Event.y - d.y();
+            this._dragging = true;
+            if (this.forceLayout) {
+                if (!d3Event.active) this.forceLayout.force.alphaTarget(0.3).restart();
```

Comment:
Restarting the alphaTarget for the optionally dragged singletons would feel like a more fluid interaction

Reply:
Nevermind, I see this is the alphaTarget for the whole simulation